cargo-concordium/src/main.rs

```diff
@@ -645,19 +645,14 @@ pub fn main() -> anyhow::Result<()> {
             only_unit_tests,
             test_targets,
         } => {
-            let success_unit = build_and_run_wasm_test(&build_options.cargo_args, seed)
-                .context("Could not build and run tests.")?;
-            let success_integration = if only_unit_tests {
-                true
-            } else {
-                build_and_run_integration_tests(build_options, test_targets).is_ok()
+            build_and_run_wasm_test(&build_options.cargo_args, seed)
+                .context("Could not build and run unit tests.")?;
```

Comment:
YOu need to remember whether any unit tests failed here, and if so raise an error at the end.